### PR TITLE
Mrbaird add hex section to sl qs guide

### PR DESCRIPTION
**Pull request by MichelleBaird-DBT**
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/guides/sl-snowflake-qs.md [hunk 1]
```diff
@@ -974,6 +974,81 @@ https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_sl-co
 
 <ConnectQueryAPI/>
 
+## Connect and query with Hex
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]: hey @MichelleBaird-DBT , what do you think of having a single 'connect and query' section? and in that section, we have diff sub-sections for diff tools? Wondering if this approach allows us to easily add more tools in the future without creating more redundant top-level sections?

Comment (mirnawong1) [reply website/docs/guides/sl-snowflake-qs.md]: Screenshot 2024-07-18 at 15 17 28

**Comment by MichelleBaird-DBT** [comment]
last minute change to assignment!

**Review by mirnawong1 — APPROVED** [review]
merging for @MichelleBaird-DBT as we need to have this up by eod today

File: website/docs/guides/sl-snowflake-qs.md [hunk 2]
```diff
@@ -965,7 +965,18 @@ https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_new-s
...
 
-## Connect and query with Google Sheets
+## Query the Semantic Layer
+
+This page will guide you on how to connect and use the following integrations to query your metrics:
+
+- [Connect and query with Google Sheets](#connect-and-query-with-google-sheets)
+- [Connect and query with Hex](#connect-and-query-with-hex)
+
+The dbt Semantic Layer enables you to connect and query your metric with various available tools like Google Sheets, Hex, and more.
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]:
```suggestion
The dbt Semantic Layer enables you to connect and query your metric with various available tools like Google Sheets, Hex, Tableau, and more. 
```

**Pull request merged**